cmd/argocd-repo-server/commands/argocd_repo_server.go

```diff
@@ -35,33 +35,16 @@ import (
 
 const (
 	// CLIName is the name of the CLI
-	cliName         = "argocd-repo-server"
-	gnuPGSourcePath = "/app/config/gpg/source"
-
-	defaultPauseGenerationAfterFailedGenerationAttempts = 3
-	defaultPauseGenerationOnFailureForMinutes           = 60
-	defaultPauseGenerationOnFailureForRequests          = 0
```

Comment:
Could we leave these as named constants and still have access to them below? They're implicit in the `var` block, but having them on their own lines is a bit more readable.